packages/d3fc-webgl/src/buffers/vertexConstantAttributeBuilder.js

```diff
@@ -7,12 +7,12 @@ export default () => {
     let data = null;
     let validSize = 0;
```

Comment:
Not necessarily part of this PR but can we change this to use `validSize` from `baseAttributeBuilder`, the same way that `elementConstantAttributeBuilder` does

Reply:
Sure